src/app/common/hooks/balance/use-stx-balance.ts

```diff
@@ -0,0 +1,17 @@
+import { baseCurrencyAmountInQuote } from '@app/common/money/calculate-money';
+import { i18nFormatCurrency } from '@app/common/money/format-money';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+import { useStacksAnchoredCryptoCurrencyAssetBalance } from '@app/query/stacks/balance/crypto-asset-balances.hooks';
+
+export function useStxAssetBalance(address: string) {
+  const stxMarketData = useCryptoCurrencyMarketData('STX');
+  const { data: balance, isLoading } = useStacksAnchoredCryptoCurrencyAssetBalance(address);
+
+  return {
+    isLoading,
+    stxAssetBalance: balance,
```

Comment:
this is `Money` type yeah?

Reply:
`CryptoCurrencyAssetItem` expects `AllCryptoCurrencyAssetBalances`
in `assetBalance` prop:
`assetBalance: AllCryptoCurrencyAssetBalances;`

that's why I used that type here